ui/cypress/plugins/index.js

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
```

Comment:
we should better stick to typescript, should be rathe changed to `index.ts` and adjusted accordingly

Reply:
File extension changed to .ts and its content adjusted accordingly.